services/simcore/docker-compose.deploy.yml

```diff
@@ -57,12 +57,16 @@ services:
         window: 120s
 
   director:
+    secrets:
+      - source: rootca.crt
+        target: /usr/local/share/ca-certificates/osparc.crt
```

Comment:
is this ok to be public?

Reply:
The certificates itself is not push in the repo, it's only when it is generated that it will be available, so I think yes ?